products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsMainWindowPresenter.cpp

```diff
@@ -2,18 +2,26 @@
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "BGSoundsMainWindowPresenter.hpp"
-#include "models/BGSoundsRepository.hpp"
-#include <application-music-player/models/SongsRepository.hpp>
+#include <apps-common/models/SongsRepository.hpp>
+#include <purefs/filesystem_paths.hpp>
 
 namespace app::bgSounds
 {
-    BGSoundsMainWindowPresenter::BGSoundsMainWindowPresenter(std::shared_ptr<AbstractSoundsRepository> soundsRepository)
+    auto bgSoundsPath = purefs::dir::getCurrentOSPath() / "assets" / "audio" / "bell" / "bg_sounds";
+
+    BGSoundsMainWindowPresenter::BGSoundsMainWindowPresenter(
+        std::shared_ptr<app::music::AbstractSongsRepository> soundsRepository)
         : soundsRepository{std::move(soundsRepository)}
-    {
-        this->soundsRepository->scanMusicFilesList();
-    }
+    {}
     void BGSoundsMainWindowPresenter::loadAudioRecords()
     {
-        getView()->setSoundsList(soundsRepository->getMusicFilesList());
+        soundsRepository->getMusicFilesList(
+            0,
+            100,
```

Comment:
what are 0 and 100? Please name the values

Reply:
Fixed. Moved to constexpr in anonymous namespace.